app/src/main/res/layout/activity_fragment_container.xml

```diff
@@ -1,7 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/fragment_container"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    />
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
My rationale here was to support the toolbar on activity level rather than on the fragment level. The coordinator layout isn't necessary though, and the toolbar can also be defined in the fragment layout, but I feel like it's more intuitively linked to the activity. What do y'all think? @gsintrepid @jmccormack200